fe/fe-core/src/main/java/org/apache/doris/analysis/InsertStmt.java

```diff
@@ -667,7 +667,7 @@ private void analyzePlanHints(Analyzer analyzer) throws AnalysisException {
     }
 
     public void prepareExpressions() throws UserException {
-        List<Expr> selectList = Expr.cloneList(queryStmt.getBaseTblResultExprs());
+        List<Expr> selectList = Expr.cloneList(queryStmt.getResultExprs());
```

Comment:
can you add some comment to explain why ?

Reply:
I'm not sure this is right, but this can solve the issues :)
I don't know why using getBaseTblResultExprs rather than getResultExprs there, are there some other considerations?

Reply:
I think that generating InserStmt's resultExprs from queryStmt's resultExprs is more reasonable, or maybe I make a mistake.

Reply:
The ```ResultExprs``` is the finally expr of querystmt.
So, this change is correct.